<launch>
  <arg name="map_topic" default="map"/>
  <arg name="map_jpeg_topic" default="map_jpeg"/>
  <arg name="publish_rate" default="0.2"/>

  <node pkg="ftt_ros_interface" type="map_to_jpeg" name="map_to_jpeg" output="screen">
    <remap from="map" to="$(arg map_topic)"/>
    <remap from="map_jpeg" to="$(arg map_jpeg_topic)"/>

    <param name="publish_rate" value="$(arg publish_rate)"/>
  </node>
</launch>